tests/test_modeling_common.py

```diff
@@ -971,3 +974,67 @@ def test_top_k_top_p_filtering(self):
 
         self.assertTrue(torch.allclose(non_inf_expected_output, non_inf_output, atol=1e-12))
         self.assertTrue(torch.all(torch.eq(non_inf_expected_idx, non_inf_idx)))
+
+    @require_torch
+    def test_postprocess_next_token_scores(self):
```

Comment:
I'm worried that this is going to run for every model. Maybe it should be in it's own `test_generation_utils.py` file?